src/toast/RELEASE

```diff
@@ -0,0 +1 @@
+2.3.7a0
```

Comment:
is this a version id? I admit I haven't checked at https://semver.org/, but to my lazy self this seems to not be a semver.

Reply:
This must be a PEP440 compliant version, not a semantic version.  When installing with setup.py (install or bdist_wheel), a version based on the git tags is used.  When running "pip wheel" it looks like git is not available, so this file contains the tag version to use instead.

Reply:
I will add a section to the docs about procedure for making a release, which (unfortunately) involves editting this file.